grails-app/conf/BuildConfig.groovy

```diff
@@ -118,8 +117,11 @@ grails.project.dependency.resolution = {
         compile ("fr.opensagres.xdocreport:org.odftoolkit.odfdom.converter.pdf:1.0.6")
         compile "org.apache.xmlgraphics:batik-util:1.7"
 
-        // Fake SMTP server
-        test 'dumbster:dumbster:1.6'
+        /*
+         * This test SMTP client is the latest release that works with Grails 1,
+         * and Java 7, although it depends on a junit release we can't use (yet).
+         */
+        test("com.icegreen:greenmail:1.5.10") { excludes "junit" }
```

Comment:
Was there a reason we changed the dependency?

Reply:
1. dumbster doesn't seem to work in JDK 1.8 (it's commented out in obgm3)
2. dumbster hasn't seen development for six years so it probably never will
3. greenmail has a straightforward upgrade path to JDK 1.8 for grails 3+ migration
4. greenmail's better documented and (imho) has a cleaner API